common.sh

```diff
@@ -124,6 +124,10 @@ fi
 
 export OPENSHIFT_RELEASE_TAG=$(echo $OPENSHIFT_RELEASE_IMAGE | sed -E 's/[[:alnum:]\/.-]*release.*://')
 
+# Both utils.sh and 04_setup_ironic.sh use this log file, so set the
+# name one time. Users should not override this.
+export MIRROR_LOG_FILE=${REGISTRY_DIR}/${CLUSTER_NAME}-image_mirror-${OPENSHIFT_RELEASE_TAG}.log
```

Comment:
The PR description says we want this to be cleaned between runs, but in the default `make clean all` case the registry dir is not cleaned (deliberately, so we can cache container images between runs).

Would it be better to put this in either `$OCP_DIR` (which is already cleaned between runs), or perhaps `$WORKING_DIR/$CLUSTER_NAME` (which is created in common.sh but AFAICS isn't cleaned between runs, but we could add cleaning it?)

Reply:
The phrasing on that message was poor. We only want the file removed when we do clean the registry, and we always want it removed when we clean the registry. I've updated the text.

Reply:
But we could switch to a different mirrored release between test runs, even if we don't clean the registry?

I guess we'll overwrite the mirror log file in that case but I just want to make sure only cleaning this on registry clean is what we want?

Reply:
The version is included in the filename via $OPENSHIFT_RELEASE_TAG so we get a new file for each version and for each cluster. That lets you install different versions into different clusters and mirror them all from the same registry instance.